test/infer/test_compute_downstream_costs.py

```diff
@@ -191,8 +242,16 @@ def test_compute_downstream_costs_big_model_guide_pair(
         model_trace, guide_trace, non_reparam_nodes
     )
 
+    dc_provenance, dc_nodes_provenance = _provenance_compute_downstream_costs(
+        model_trace, guide_trace, partial(_get_log_probs, _get_traces)
+    )
+
     assert dc_nodes == dc_nodes_brute
 
+    for name, nodes in dc_nodes_provenance.items():
+        assert nodes.issubset(dc_nodes[name])
+        assert nodes == {name}
```

Comment:
Since there are no dependencies in the test model/guide it just turns out that `nodes == {name}` for provenance based dependency tracking.